pandas/io/formats/terminal.py

```diff
@@ -48,6 +48,23 @@ def get_terminal_size():
     return tuple_xy
 
 
+def is_terminal():
+    """
+    Detect if Python is running in a terminal.
+
+    Returns True if Python is running in a terminal or False if not.
+    """
+    try:
+        ip = get_ipython()
+    except NameError:  # assume standard Python interpreter in a terminal
```

Comment:
don’t you need to import this?

Reply:
No, this is available in IPython.

Reply:
I.e. when you run IPython `get_ipython` is automatically imported.

Reply:
got it, this is ok then.